### PR TITLE
docs: add DEP user flow spec

### PR DESCRIPTION
**Pull request by youngjoon-lee**
As a frontpage of all spec docs.

### CODE REVIEW

**Review by gyuguen — APPROVED** [review]
LGTM
I think the flow of content seems to be fine.

File: .gitbook/3-protocol-devs/1-dep-specs/1-user-flow.md [hunk 1]
```diff
@@ -13,23 +13,68 @@
...
 ## Technical Specification
 
-### User Flow
+![](images/user-flow.drawio.png)
+
+The diagram describes each key player interacts with each other to complete a cycle of data exchange. The whole process consists of several steps below.
+
+### Data Deal Registration
+
+A data consumer creates a data deal, specifying data schema, a budget in MED, the maximum number of data to consume, and agreement terms. The data consumer should publish the data deal to a public and secure environment, such as a public blockchain, to ensure that the data deal is not changed by anyone else after it is created. In this document, the Panacea blockchain is used for that public environment. For more details of the data deal, please see the [Data Deal](2-data-deal.md) specification.
```
Comment (H4NLee) [inline .gitbook/3-protocol-devs/1-dep-specs/1-user-flow.md:52]: In the far future, can someone create a deal with other cryptocurrency, not MED?
I have imagined 😂

**Review by H4NLee — APPROVED** [review]
Clear.
Based on this doc, we can refine and make cleaner other spec docs.

File: .gitbook/3-protocol-devs/1-dep-specs/1-user-flow.md (hunk 1 above)
Comment (youngjoon-lee) [reply .gitbook/3-protocol-devs/1-dep-specs/1-user-flow.md:52]: I think, yes :)

**Review by audtlr24 — APPROVED** [review]
LGTM

**Review by inchori — APPROVED** [review]
LGTM!

**Pull request merged**